pages/home.page.tsx

```diff
@@ -78,33 +82,78 @@ export default function HomePage({ navigation }: IReactPageServices) {
+        <IconIonicons name="settings-outline" color={colors.white} size={24} />
+    
+      </View>
       )
     });
   });
 
   return (
     <Page >
-      <StatusBar style="auto" />
-      <View style={{ width: "100%" }} >
-        <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center" }]} source={require('../assets/app-icon.png')} />
-        <Text style={[{ textAlign: 'center', marginBottom: 5, color: themePalette.shellTextColor, fontSize: 24 }]}>{user?.currentOrganization.text} Instances</Text>
+      {/* <StatusBar style="auto" /> */}
+      <View style={{padding: 16, width: "100%", height: "100%", backgroundColor: themePalette.background }} >
+        {/* <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center" }]} source={require('../assets/logo.png')} /> */}
+        {themePalette.name == 'light' ? (
+          <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center" }]} source={require('../assets/logo.png')} />
+        ) : (
+          <Image style={[{ marginTop: 30, marginBottom: 30, alignSelf: "center"}]} source={require('../assets/logo-dark-theme.png')} />
+        )}
         <FlatList
-          contentContainerStyle={{ alignItems: "stretch" }}
-          style={{ backgroundColor: themePalette.background, width: "100%", flexGrow: 1 }}
+          contentContainerStyle={{ height: "auto"  }}
+          style={{ backgroundColor: themePalette.background, width: "100%" }}
           ItemSeparatorComponent={myItemSeparator}
           ListEmptyComponent={myListEmpty}
           data={instances}
           renderItem={({ item }) =>
             <Pressable onPress={() => showInstance(item)} key={item.id} >
-              <View style={[styles.listRow, { padding: 10, marginLeft:10, marginRight:10, marginBottom: 10, height: 60, backgroundColor: themePalette.inputBackgroundColor, }]}  >
-                <SLIcon icon={item.icon} />
-                <Text style={[{  marginLeft:10,  marginTop: 3, color: themePalette.shellTextColor, fontSize: 24, flex: 3 }]}>{item.name}</Text>
+              <View style={[styles.listRow, { display:"flex", alignItems:"center", gap:16, padding: 8, margin: 8, height: 72, backgroundColor: themePalette.inputBackgroundColor, shadowColor: "#1F2F4C", shadowOffset: { width: 0, height: 2 }, shadowOpacity: 0.2, shadowRadius: 2, borderRadius: 8 }]}  >
+              
+                {/* <SLIcon icon={item.icon} /> */}
+                {/* <Image source={require('../assets/product-img.png')} /> */}
+                <View
+                      style={{
+                        backgroundColor: "#3377FF",
+                        borderRadius: 8,
+                        height: 56,
+                        width: 56,
+                        alignItems: "center",
+                        justifyContent: "center",
+                      }}
+                    >
+                      <IconFeather
```

Comment:
We should try to pickup the icon from item.icon rather than have "cpu" hardcoded.